modules/terraform/cmd.go

```diff
@@ -20,6 +20,12 @@ func generateCommand(options *Options, args ...string) shell.Command {
 	return cmd
 }
 
+var commandsWithParallelism = []string{
+	"plan",
+	"apply",
+	"destroy",
```

Comment:
This should include `plan-all` and `apply-all` for terragrunt.

Reply:
Good catch! https://github.com/gruntwork-io/terratest/pull/572